### DISCUSSION TITLE
How does the magic of <UseMaui>true</UseMaui> work?

### DISCUSSION THREAD

**Discussion by voroninp** [dotnet/maui#7456] [Q&A] [answered]
I'd like to know what happens under the hood with MSBuild and VS when they see this flag on.

**Comment by Dreamescaper** [comment, marked as answer]
It's done by a bunch of .targets files. Take a look here, for instance

https://github.com/dotnet/maui/blob/main/src/Workload/Microsoft.Maui.Sdk/Sdk/Sdk.targets
